components/director/internal/domain/api/resolver.go

```diff
@@ -156,6 +169,9 @@ func (r *Resolver) DeleteAPIDefinition(ctx context.Context, id string) (*graphql
 		return nil, err
 	}
 	defer r.transact.RollbackUnlessCommitted(tx)
+
+	log.Infof("Deleting APIDefinition %s", id)
```

Comment:
maybe in these logs you could clarify that you are logging the id - "Deleting APIDefinition with id %s"

Reply:
Agreed.

Actually, most of the existing logs prior to this PR were in this format without explicitly mentioning `with id`. I followed this approach for consistency reasons but as it was pointed out in the other comments - it is way more useful using the `with id` format.

So, there are still some leftover logs with the old format, I will go through all of them and add it.